uploadservice-s3/src/main/java/net/gotev/uploadservice/s3/S3Setup.kt

```diff
@@ -0,0 +1,48 @@
+package net.gotev.uploadservice.s3
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferService
+import java.util.UUID
+
+class S3Setup(context: Context) {
```

Comment:
This branch is `feature/box upload`. How does this S3 work relate?

Reply:
Alright I made a separate branch and PR for these fixes. If we merge that first, these will be gone from this branch.